src/methods/defining_methods/hard/q6/IntegerMulter.java

```diff
@@ -0,0 +1,31 @@
+package methods.defining_methods.hard.q6;
+
+/*
+Write a Java program that asks users for an integer between 0 and 10 (inclusive), which has a method called multiply1000
```

Comment:
Change question text to:
`Write a Java program that asks users for an integer between 0 and 10 (inclusive) and then multiples it by 1000. To multiply the number, create a  method called multiply1000 that takes an integer as an input parameter and returns that integer multiplied by 1000. If the method receives an invalid integer ( less than 0 or greater than 10), return -1. Call the method with the user input and print the returned integer in the main method.`